### PR TITLE
Update to temporary maintenance branch

### PR DESCRIPTION
**Pull request by Diegovsky**

### CODE REVIEW

File: dev.alextren.Spot.json [hunk 1]
```diff
@@ -36,6 +36,17 @@
         "*.a"
     ],
     "modules": [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
```
Comment (bbhtt) [inline dev.alextren.Spot.json:42]: This should be cleaned up, it's only needed at build time `"cleanup": ["*"]`

Comment (Diegovsky) [reply dev.alextren.Spot.json:42]: Sure!

**Comment by Diegovsky** [comment]
@bbhtt I have a question regarding the `custom-key-duplicated` lint. How should I specify Spot supports both mobile and desktop environments?

**Comment by Diegovsky** [comment]
bot, build

**Comment by bbhtt** [comment]
> @bbhtt I have a question regarding the `custom-key-duplicated` lint. How should I specify Spot supports both mobile and desktop environments?

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-relations-control, touch will make it show up as supported on mobile I think. You can view the metainfo file with `gnome-software --show-metainfo /path/to/appid.metainfo.xml`

**Comment by Diegovsky** [comment]
bot, build

**Comment by Diegovsky** [comment]
oof, forgot to update the tag. might start using a branch instead

bot, build

**Comment by Diegovsky** [comment]
finally!

**Comment by Diegovsky** [comment]
thank you @bbhtt for your patience and knowledge!

**Comment by AnhQuanTrl** [comment]
Thank you @Diegovsky for stepping up to maintain this awesome package!

**Pull request merged**